code/game/jobs/job/civilian.dm

```diff
@@ -249,6 +251,7 @@
 	selection_color = "#dddddd"
 	access = list(access_library, access_maint_tunnels)
 	minimal_access = list(access_library)
+	alt_titles = list("Curator", "Archivist")

```

Comment:
What is the difference between the librarian, the curator and the archivist ?

Reply:
Xenoarch interaction?

Reply:
I mean, I could just remove the archivist and leave only curator as alternative